Doc-11534-PM-2.8-Edits-DH  Partner Manager 2.8 edits (#141)
mulesoft/docs-partner-manager: modules/ROOT/Edits/modify-third-party-settings.adoc

> = Modify a 3rd Party Connection's Settings

Reviewer: ```suggestion
= Modify 3rd Party Connection Settings
```

> You can modify 3rd party connection settings including contact details, identifiers, digital certificates, and more.

Reviewer: ```suggestion
You can modify 3rd party connection settings such as contact details, identifiers, and digital certificates.
```

To modify 3rd party connection settings:

. In Partner Manager, select *Partners* in the navigation menu.
. Click the connection you want to modify.

> . Modify any of the following settings:

Reviewer: ditto re. substeps in table
